src/storage/NamedSetList.ts

```diff
@@ -132,26 +133,29 @@ export default class NamedSetList {
           return;
         }
 
-        const deleteIt = await areyousure(`The named set <i>${namedSet.name}</i> will be deleted and cannot be restored. Continue?`,
-          {title: `Delete named set`}
+        const deleteIt = await areyousure(i18n.t('tdp:core.NamedSetList.dialogText', {name: namedSet.name}),
+          {title: i18n.t('tdp:core.NamedSetList.deleteSet')}
         );
         if (deleteIt) {
           await deleteNamedSet(namedSet.id);
-          successfullyDeleted('Dashboard', namedSet.name);
+          successfullyDeleted(i18n.t('tdp:core.NamedSetList.dashboard'), namedSet.name);
           this.remove(namedSet);
         }
       });
 
     //update
     $options.select('a.goto').text((d) => d.name)
-      .attr('title', (d) => `Name: ${d.name}\nDescription: ${d.description}${d.type === ENamedSetType.NAMEDSET ? `\nCreator: ${(<IStoredNamedSet>d).creator}\nPublic: ${hasPermission(<IStoredNamedSet>d, EEntity.OTHERS)}` : ''}`);
+      .attr('title', (d) => {
+        const extendedData = d.type === ENamedSetType.NAMEDSET ? {context: 'extended', creator: (<IStoredNamedSet>d).creator, public: hasPermission(<IStoredNamedSet>d, EEntity.OTHERS)} : {};
+        return i18n.t('tdp:core.NamedSetList.title', {name: d.name, description: d.description, ...extendedData}) as string; // i18next context feature
+      });
     $options.select('a.delete').classed('hidden', (d) => d.type !== ENamedSetType.NAMEDSET || !canWrite(d));
     $options.select('a.edit').classed('hidden', (d) => d.type !== ENamedSetType.NAMEDSET || !canWrite(d));
     $options.select('a.public')
       .classed('hidden', (d) => d.type !== ENamedSetType.NAMEDSET || !canWrite(d))
       .html((d) => {
         const isPublic = d.type === ENamedSetType.NAMEDSET && hasPermission(<IStoredNamedSet>d, EEntity.OTHERS);
-        return `<i class="fa ${isPublic ? 'fa-users' : 'fa-user'}" aria-hidden="true" title="${isPublic ? 'Public' : 'Private'}"></i> <span class="sr-only">${isPublic ? 'Public' : 'Private'}</span>`;
+        return `<i class="fa ${isPublic ? 'fa-users' : 'fa-user'}" aria-hidden="true" title="${i18n.t('tdp:core.NamedSetList.status', {context: isPublic ? '' : 'private'})}"></i> <span class="sr-only">${isPublic ? 'Public' : 'Private'}</span>`;
```

Comment:
You missed the string _Public_ and _Private_.

Reply:
Thank you for the feedback.